src/frequenz/sdk/timeseries/battery_pool/battery_pool.py

```diff
@@ -447,6 +454,28 @@ def capacity(self) -> ReceiverFetcher[Sample[Energy]]:
 
         return self._active_methods[method_name]
 
+    def power_bounds(
+        self, priority: int = 0
+    ) -> ReceiverFetcher[_power_managing.Report]:
+        """Get a receiver to receive new power bounds when they change.
+
+        These bounds are the bounds specified the power manager for actors with the
+        given priority.
```

Comment:
Can you add more words to this sentence so it doesn't read like I am hiking barefoot through a field of sharp stones?

Reply:
:D wow, that's so funny, the stuff I've written.  But already fixed.  And doing the other doc stuff done as well.